db/internal_stats.cc

```diff
@@ -152,6 +153,8 @@ void PrepareLevelStats(std::map<LevelStatType, double>* level_stats,
       static_cast<double>(stats.num_dropped_records);
   (*level_stats)[LevelStatType::R_BLOB_GB] = stats.bytes_read_blob / kGB;
   (*level_stats)[LevelStatType::W_BLOB_GB] = stats.bytes_written_blob / kGB;
+  (*level_stats)[LevelStatType::PB_COMP_COUNT] =
+      stats.preclude_bottommost_count;
```

Comment:
What does it exactly mean? Count of compactions?

I feel that if we turn 2-level compaction only happens to the **last** level, and not **botommost** level, stats are going to be simpler here and maybe we don't need this new column.

Reply:
Removed, as we switched to use `preclude_last_level`.